### ISSUE TITLE
Includes TLDs that are not registerable (.kerryproperties)

### ISSUE THREAD

**Issue by Cobertos** [Cobertos/tld-data#1]
For some reason `.kerryproperties` is included in the dataset, when you definitely can't register it. (`nic.kerryproperties` does not resolve and the sponsoring party is a real-estate company, `kerryprops.com`). Most sites say it's a "brand TLD":

* It's listed on [`ntldstats.com`](https://ntldstats.com/tld/kerryproperties) as a brand TLD.
* It's listed as a Brand TLD on [ICANN Wiki](https://icannwiki.org/.kerryproperties).
* Listed as an "Organization" on [tld-list](https://tld-list.com/tld/kerryproperties) though so is `.dev` so perhaps it also has trouble classifying
* It's listed as a "Generic brand" TLD on [namestat](https://namestat.org/kerryproperties#keyinfo) and I have yet to look at where they're scraping data from.

Yet...

* I see no mention of Specification 13 in any of the [ICANN registry agreements](https://www.icann.org/resources/agreement/kerryproperties-2015-04-09-en)
  * Not in the original registry agreement
  * Not in any amendment
  * There's no "Specification 13" section on the registry agreement page

**Comment by Cobertos** [comment]
Similar issues with `.akdn`

**Comment by Cobertos** [comment]
ICANN Wiki has a big list here https://icannwiki.org/New_gTLD_Brand_Applications

But all the sources are dead...

Could just scrape that but it's annoying it's not coming from an official source.

**Comment by Cobertos** [comment]
Domainr returns nothing of interest for this.

**Comment by Cobertos** [comment]
After scrapping Wikipedia and comparing to my data, I found quite a few issues with Wikipedia's data, at least when it came to Specification 13.

I noticed for many of the ones that Wikipedia listed that mine didn't, there was a Specification 9 exemption. Low and behold that at the bottom of Specification 9 it specifically mentions a Registry registering only using it for themselves and direct affiliates.

So it looks to be that the presence of a Specification 9 exemption means it's a brand TLD.

Some like `.dev` used to have such an exemption but have now been withdrawn. Would explain some issues on some sources with TLDs like `.bond` and `.dev`.
